src/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -24,9 +24,13 @@
     <Compile Include="System\DirectoryServices\BinaryCompatibility.cs" />
     <Compile Include="System\DirectoryServices\DerefAlias.cs" />
     <Compile Include="System\DirectoryServices\DirectoryEntries.cs" />
-    <Compile Include="System\DirectoryServices\DirectoryEntry.cs" />
+    <Compile Include="System\DirectoryServices\DirectoryEntry.cs">
+      <SubType>Component</SubType>
+    </Compile>
```

Comment:
I don't know - it was something the tooling did.    I reverted the changes and the code is still building fine w/o them.